data/is_ocaml_yet/gui.md

```diff
@@ -0,0 +1,31 @@
+---
+id: gui
+question: Is OCaml GUI yet?
+answer: Some hidden gems, we just need to a little bit of digging!
+categories:
+  - name: Bindings
+    status: 🆘
+    description: |
+      n/a
+    packages: []
+  - name: Cross-Platform
+    status: 🆘
+    description: |
+      n/a
+    packages: []
+  - name: Immediate Mode GUI
+    status: 🆘
+    description: |
+      n/a
+    packages: []
+  - name: Platform-Specific
+    status: 🆘
+    description: |
+      n/a
+    packages: []
+  - name: Web Technologies
+    status: 🆘
+    description: |
+      n/a
+    packages: []
```

Comment:
I suggest to add : 
- [Bonsai](https://github.com/janestreet/bonsai)
- [OCaml-vdom](https://github.com/LexiFi/ocaml-vdom)
- [Fmlib_browser](https://github.com/hbr/fmlib)

Reply:
I added these in the "Browser" category in "Is OCaml Web Yet?". I think the intention of Web Technologies for GUI should be more along the line of JavaScript engines, or HTML/CSS rendering engines

Reply:
For these packages, a better category than "Browser" would be "Virtual DOM" as this is a keyword known in the frontend web dev space.